grpc-gcp/src/main/java/com/google/cloud/grpc/GcpMultiEndpointChannel.java

```diff
@@ -236,8 +239,9 @@ public void setMultiEndpoints(List<GcpMultiEndpointOptions> meOptions) {
                 .build());
       }
     });
-    // Must have all multiendpoints before initializing the pools so that all multiendpoints
-    // can get status update of every pool.
+
+    // TODO: Add support for different credentials for the same endpoint.
```

Comment:
Let's make this more specific, i.e.:
// TODO: Support the same endpoint in different MultiEndpoint to use different channel credentials.
// TODO: Support different endpoints in the same MultiEndpoint to use different channel credentials.

Reply:
SGTM. Done.